<?xml version="1.0"?>
<launch>
    <group ns="lidar_0">
        <param name="mode" value="single"/>
        <param name="input_topic" value="/lidar_0/lidar_node/pointcloud"/>
        <param name="output_topic" value="/lidar_0/pointcloud_filtered"/>

        <node pkg="multi_lidar_calibration" type="cloud_cropper_node" name="cloud_cropper_node" output="screen"/>
        <node pkg="multi_lidar_calibration" type="cloud_crop_helper.py" name="cloud_crop_help" output="screen"/>
    </group>
</launch>